src/components/DetailsInfo/detailsInfo.util.js

```diff
@@ -206,13 +205,19 @@ export const generateAlertsDetailsInfo = pageData => {
       notificationsDetailsInfo: [],
       triggerCriteriaDetailsInfo: []
     }
-    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((valueItem, index) => (
-      <Tooltip
-        key={valueItem.tooltip + index}
-        template={<TextTooltipTemplate text={valueItem.tooltip} />}
-      >
-        {valueItem.icon}
-      </Tooltip>
+    AlertsDetailsInfo.notificationsDetailsInfo = notifications.map((notification, index) => (
+      <li className="notifications-item" key={index}>
+        <div className="notifications-item_icon">{notification.icon}</div>
+        <div>
+          <div className="notifications-item_header">{upperFirst(notification.kind)}</div>
+          <div className="notifications-item_header-text">
+            {`${notification.succeeded} success `}
+            <span className={notification.failed > 0 ? 'text-red' : ''}>
```

Comment:
use classnames library, also rename text-red to notifications-item__header-text_failed

Reply:
Done